### ISSUE TITLE
esim  question

### ISSUE THREAD

**Issue by ZXsilence** [pengming617/text_matching#12]
```
 # 计算a_bar与b_bar每个词语之间的相似度
            with tf.name_scope('word_similarity'):
                attention_weights = tf.matmul(a_bar, tf.transpose(b_bar, [0, 2, 1]))
                attentionsoft_a = tf.nn.softmax(attention_weights)
                attentionsoft_b = tf.nn.softmax(tf.transpose(attention_weights))
                attentionsoft_b = tf.transpose(attentionsoft_b)
                a_hat = tf.matmul(attentionsoft_a, b_bar)
                b_hat = tf.matmul(attentionsoft_b, a_bar)
```

我感觉这段代码里面 attentionsoft_b 的代码逻辑好像不太对 我认为的逻辑是
` attentionsoft_b = tf.nn.softmax(tf.transpose(attention_weights, [0, 2, 1]))`

**Comment by pengming617** [comment]
嗯嗯  确实写错了  谢谢你得指出
